package.json

```diff
@@ -294,7 +294,7 @@
 		"@automattic/webpack-inline-constant-exports-plugin": "file:./packages/webpack-inline-constant-exports-plugin",
 		"@types/classnames": "2.2.9",
 		"@types/debug": "4.1.4",
-		"@types/lodash": "4.14.123",
+		"@types/lodash": "4.14.144",
```

Comment:
Interesting, renovate didn't update the description, but this is the current latest.